src/components/learner-credit-management/data/hooks/useOfferRedemptions.js

```diff
@@ -69,14 +78,27 @@ const useOfferRedemptions = (enterpriseUUID, offerId = null, budgetId = null) =>
           applySortByToOptions(args.sortBy, options);
         }
         if (args.filters?.length > 0) {
-          applyFiltersToOptions(args.filters, options);
+          applyFiltersToOptions(args.filters, options, fetchTransactions);
         }
-        const response = await EnterpriseDataApiService.fetchCourseEnrollments(
-          enterpriseUUID,
-          options,
-        );
-        const data = camelCaseObject(response.data);
-        const transformedTableResults = transformUtilizationTableResults(data.results);
+        let data;
+        let transformedTableResults;
+        if (offerId === null && fetchTransactions) { // and do the feature flag check here
```

Comment:
nit: was a bit confused by the comment `// and do the feature flag check here`. The `fetchTransactions` variable currently represents whether the feature flag is enabled or not, correct? Perhaps make the comment more explicit by stating something along the lines of `// Check if the budget is a subsidy access policy and whether the feature flag is enabled`.

Reply:
relic of some copy pasta! Happy to rework

Reply:
it was truly nothing more than a todo lol